Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml

```diff
@@ -0,0 +1,1268 @@
+      description: |-
+        Uses Active Directory to retrieve detailed information about the computer of the account specified. The computer can be specified by name, email address, or as an Active Directory Distinguished Name (DN).
+        If no filters are specified, returns all computers.
+      script: Active Directory Query v2|||ad-get-computer
+      type: regular
+      iscommand: true
+      brand: Active Directory Query v2
+    nexttasks:
+      '#none#':
+      - "4"
+    scriptarguments:
+      name:
+        complex:
+          root: inputs.Hostname
+          transformers:
+          - operator: uniq
+    reputationcalc: 1
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -430,
+          "y": 890
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+    continueonerrortype: ""
+  "24":
+    id: "24"
+    taskid: 9706cc39-d338-44cd-8ee1-efc5ea95b04d
+    type: title
+    task:
+      id: 9706cc39-d338-44cd-8ee1-efc5ea95b04d
+      version: -1
+      name: Active Directory
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "22"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -240,
+          "y": 580
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+    continueonerrortype: ""
+  "26":
+    id: "26"
+    taskid: d973a016-35b6-4e7c-88a5-9f4c60190224
+    type: playbook
+    task:
+      id: d973a016-35b6-4e7c-88a5-9f4c60190224
+      version: -1
+      name: Endpoint Enrichment - Cylance Protect v2
+      description: Enriches endpoints using the Cylance Protect v2 integration.
+      playbookName: Endpoint Enrichment - Cylance Protect v2
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "4"
+    scriptarguments:
+      Hostname:
+        complex:
+          root: inputs.Hostname
+          transformers:
+          - operator: uniq
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 0
+    view: |-
+      {
+        "position": {
+          "x": 1480,
+          "y": 710
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "29":
+    id: "29"
+    taskid: be810018-6fbe-4194-8a08-c44cb784103d
+    type: regular
+    task:
+      id: be810018-6fbe-4194-8a08-c44cb784103d
+      version: -1
+      name: Get endpoint information from Crowdstrike Falcon
+      description: Returns information about an endpoint.
+      script: CrowdstrikeFalcon|||endpoint
```

Comment:
![image](https://user-images.githubusercontent.com/28757135/227764917-96d5c180-5800-41bf-b0c6-d92e5d9da8f0.png)
Why we use endpoint(crowdstrike) if we use Crowdstrike command and 'endpoint' has it own task?

Reply:
The endpoint task was removed.